src/kudu/consensus/consensus.proto

```diff
@@ -399,11 +406,19 @@ message ConsensusRequestPB {
   // UUID of server this request is addressed to.
   optional bytes dest_uuid = 7;
 
+  // UUID of server intended to proxy this request.
```

Comment:
is it better to have this wording "UUID of server which will proxy this request to eventual dest_uuid"?

Reply:
Yes, this is better.